bibliopixel/drivers/driver_base.py

```diff
@@ -47,7 +47,6 @@ def __init__(self, num=0, width=0, height=0, c_order=ChannelOrder.RGB,
         self.lastUpdate = 0
         self.brightness_lock = threading.Lock()
         self._brightness = 255
-        self.set_brightness(255)
```

Comment:
Since this is run before the reset of each driver's __init__() code executes it was crashing due to DriverSerial not yet being connected to any device and since it uses set_device_brightness instead. But this wasn't really necessary anyways.